addons/l10n_sa_edi/models/res_partner.py

```diff
@@ -18,7 +18,7 @@ class ResPartner(models.Model):
         ('GCC', 'GCC ID'),
         ('IQA', 'Iqama Number'),
         ('PAS', 'Passport ID'),
-        ('OTH', 'Other ID')
+        ('OTH', 'Other ID'),
```

Comment:
useless diff, let's not touch a file where we did nothing just for cosmetic even if i like the effort ahah